bench/run-rpu-sweep-phase2b-rerun.sh

```
@@ -0,0 +1,112 @@
+# at all RPU values because markers from Phase 1/2a prevented score clearing.
+#
+# This script ONLY touches Phase 2b profiles.  Per pitfall P1, we do NOT
+# modify run-per-pubkey-sweep.sh — instead this standalone script:
+#   - Clears scores PER-PROFILE using ${pk:0:16} globs (not wildcards)
+#   - Uses per-(RPU × profile) markers that cannot collide with Phase 1
+#   - Has its own progress file
+#
+# 3 profiles × 5 RPU × 5 sessions = 75 runs
+set -uo pipefail
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)